src/content/reference/react-dom/components/common.md

```diff
@@ -1,95 +1,96 @@
 ---
-title: "Common components (e.g. <div>)"
+title: "Komponen umum (e.g. <div>)"
 ---
 
 <Intro>
 
-All built-in browser components, such as [`<div>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div), support some common props and events.
+Semua komponen bawaan dari sebuah peramban web (*browser*), seperti [`<div>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div), mendukung beberapa *props* umum dan *event*. 
```

Comment:
```suggestion
Semua komponen bawaan dari sebuah peramban web (*browser*), seperti [`<div>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div), mendukung beberapa *props* dan *event* umum. 
```

Reply:
Noted and fixed, thank you for the review